tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -531,6 +576,14 @@ expressed in the logs.
 {"message": "granules table created.", "timestamp": "2021-09-14T15:20:20.231299", "level": "info"}
 {"message": "files table created.", "timestamp": "2021-09-14T15:20:20.231299", "level": "info"}
 {"message": "Data added to the schema_versions table.", "timestamp": "2021-09-14T15:20:20.231299", "level": "info"}
+{"message": "reconcile_status table created.", "timestamp": "2022-02-10T10:51:07.422447", "level": "info"}
+{"message": "reconcile_job table created.", "timestamp": "2022-02-10T10:51:07.422962", "level": "info"}
+{"message": "reconcile_s3_object table created.", "timestamp": "2022-02-10T10:51:07.423450", "level": "info"}
+{"message": "reconcile_catalog_mismatch_report table created.", "timestamp": "2022-02-10T10:51:07.424753", "level": "info"}
+{"message": "reconcile_orphan_report table created.", "timestamp": "2022-02-10T10:51:07.425242", "level": "info"}
+{"message": "reconcile_phantom_report table created.", "timestamp": "2022-02-10T10:51:07.425728", "level": "info"}
+{"message": "partition table for reconcile_s3_object created.", "timestamp": "2022-02-10T10:51:07.426223", "level": "info"}
```

Comment:
May want to update this message to provide the partition name. This should also occur after the creation of the base table.

Reply:
fixed